lms/djangoapps/certificates/management/commands/tests/test_cert_management.py

```diff
@@ -129,25 +129,24 @@ def test_course_caching(self):
         self._create_cert(self.courses[0].id, UserFactory.create(), CertificateStatuses.error)
         self._create_cert(self.courses[0].id, UserFactory.create(), CertificateStatuses.error)
 
-        # Verify that we make only one Mongo query
-        # because the course is cached.
-        with check_mongo_calls(1):
+        course_overview = CourseOverviewFactory.create(
+            id=self.courses[0].id
+        )
+
+        with patch(
+            'lms.djangoapps.certificates.management.commands.resubmit_error_certificates.get_course_overview'
+        ) as mock_get_course_overview:
+            mock_get_course_overview.return_value = course_overview
+
             call_command(self.command)
 
+            mock_get_course_overview.assert_called_once()
+
     def test_invalid_course_key(self):
         invalid_key = "invalid/"
         with self.assertRaisesRegex(CommandError, invalid_key):
             call_command(self.command, course_key_list=[invalid_key])
 
-    def test_course_does_not_exist(self):
```

Comment:
This test no longer makes sense. Our test classes do some interesting things.

The `_create_cert` function uses the `CourseEnrollmentFactory` to create a course-run enrollment for the user in the test. Automagically, behind the scenes, the `CourseEnrollmentFactory` is also creating a CourseOverview for the `course_key` provided from the test.

Thus there is no way to have the course "not exist" as the test/factory is automatically creating the CourseOverview for us.

In the end, I didn't think this was a high risk to remove, and hopefully this test and management command are removed in the near future with the rest of the v1 course certificate code.